configuration/windows-gaming-hvm.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: doc
+title: Windows gaming hvm
+permalink: /doc/windows-gaming-hvm/
+---
+
+# Windows gaming HVM
+
+Some information to configure a windows HVM for gaming. 
+This is not officially supported, just some community trial & errors
```

Comment:
I think this would be much more appropriate for https://github.com/Qubes-Community. @awokd and others can help you with this.

Reply:
Also, we need to be careful about taking on too many optional docs. The more we have, the harder they are to maintain, and the less we are able to ensure high quality. See https://github.com/QubesOS/qubes-issues/issues/4693 for more on this.

Reply:
> @awokd and others can help you with this.

Be happy to. @neowutran, I'll flag you over there.